sentry_sdk/scope.py

```diff
@@ -121,7 +122,49 @@ def __init__(self):
         self._propagation_context = None  # type: Optional[Dict[str, Any]]
 
         self.clear()
-        self.generate_propagation_context()
+
+        incoming_trace_information = self._load_trace_data_from_env()
+        self.generate_propagation_context(incoming_data=incoming_trace_information)
+
+    def _load_trace_data_from_env(self):
+        # type: () -> Optional[Dict[str, str]]
+        """
+        Load Sentry trace id and baggage from environment variables.
+        Can be disabled by setting SENTRY_USE_ENVIRONMENT to "false".
+        """
+        incoming_trace_information = None
+
+        false_values = [
+            "false",
+            "False",
+            "FALSE",
+            "no",
+            "No",
+            "NO",
+            "n",
+            "0",
+            "off",
+            "Off",
+            "OFF",
+        ]
+
+        use_environment = (
+            False if os.environ.get("SENTRY_USE_ENVIRONMENT") in false_values else True
```

Comment:
Could we have `os.environ.get("SENTRY_USE_ENVIRONMENT")` in a variable and do a lowercase transformation on it? Then we would not have to maintain a list of possible lower/uppercase combinations of `false`, `no`, `off` and so on (see `false_values` array)

Reply:
good point. yes we can!